react/components/organisms/chat/chat.js

```diff
@@ -117,6 +123,7 @@ export class Chat extends PureComponent {
                     })}
                 </ScrollView>
                 <RichTextInput
+                    ref={el => (this.richTextInputComponent = el)}

```

Comment:
Let's simplify: richTextInputComponent -> input. In the context of the chat it's not very relevant what's the variant of the input it is using, only that it is an input.